### DISCUSSION TITLE
Is it possible to use diagonal arrows in the `CD` environment?

### DISCUSSION THREAD

**Discussion by Ahsoka** [KaTeX/KaTeX#3033] [Q&A] [answered]
Perhaps something like this?

![Image for reference](https://i.stack.imgur.com/22hgl.png)

Thank you!

**Comment by edemaine** [comment, marked as answer]
Sadly, I don't think so. See [amscd documentation](https://ctan.math.illinois.edu/macros/latex/required/amsmath/amscd.pdf)

**Reply by Ahsoka** [reply]
Thank you for the prompt response and for the documentation as well! This is very helpful, could this feature perhaps be added in a future release of KaTeX?

Thank you!

**Reply by edemaine** [reply]
Not through the `{CD}` environment, but LaTeX offers other environments that KaTeX could try to add, probably as extensions. The ones I know of (previously requested) are `xymatrix` (#1834) and `tikz` (#219, very hard). If you know of other LaTeX environments that might be easier to implement, I'd be curious to hear them.
